frontend/components/SetupPage/Create/SetupEoaFunding.tsx

```diff
@@ -117,10 +144,78 @@ export const SetupEoaFundingForChain = ({
   );
 };
 
+type SetupEoaFundingPropsV2 = {
+  isFunded: boolean;
+  minRequiredBalance: number;
+  currency: string;
+  chainName: string;
+};
+
+const SetupEoaFundingForChainV2 = ({
+  isFunded,
+  minRequiredBalance,
+  currency,
+  chainName,
+}: SetupEoaFundingPropsV2) => (
+  <>
+    <Paragraph style={{ marginBottom: 12 }}>
+      Send funds on Base to create your account. Additional funds for staking
+      and operating your agent will be requested separately.
+    </Paragraph>
+
+    <CardSection>
+      <CustomAlert
+        fullWidth
+        type="warning"
+        showIcon
+        message={
+          <Flex vertical gap={5}>
+            <Text strong>Only send funds on {chainName} chain!</Text>
+            <Text>You will lose any assets you send on other chains.</Text>
+          </Flex>
+        }
+      />
+    </CardSection>
+    <WaitingForTransactionCard className="mt-16">
+      <Flex gap={8} align="center">
+        <Image
+          src={`/chains/${kebabCase(chainName)}-chain.png`}
+          width={20}
+          height={20}
+          alt="chain logo"
+        />
+        <Text>{chainName}</Text>
+      </Flex>
+      <Flex
+        justify="space-between"
+        align="center"
+        gap={16}
+        style={{ width: '100%' }}
+      >
+        <Text strong>
+          {minRequiredBalance} {currency}
+        </Text>
+        <Line />
+        <Text
+          style={{ display: 'block', width: 202 }}
+          type="secondary"
+          className={isFunded ? '' : 'loading-ellipses'}
+        >
+          {statusMessage(isFunded)}
+        </Text>
+      </Flex>
+    </WaitingForTransactionCard>
+    <AccountCreationAddress />
+  </>
+);
+
+type SendFundAction = 'native' | 'nonNative';
```

Comment:
a bit unclear what is native/nonNative. sounds like it's more about token
```suggestion
type SendFundAction = 'transfer' | 'bridge';
```
wdyt?